traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -242,6 +242,30 @@ func getServers(params map[string]string, tx *sqlx.Tx, user *auth.CurrentUser) (
 		"dsId":             dbhelpers.WhereColumnInfo{"dss.deliveryservice", nil},
 	}
 
+	searchQueryParamsToSQLCols := map[string]dbhelpers.WhereColumnInfo{
```

Comment:
We should make sure all the columns we're allowing search on have indexes, to prevent performance issues and accidental or intentional DDOS